hack/make-release-artifacts.sh

```diff
@@ -46,7 +46,7 @@ done < <(find "${bin_dir}" -print0)
 echo >&2 "Creating ${krew_tar_archive} archive."
 (
   cd "${bin_dir}"
-  GZIP=-n tar czvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
+  tar -I "gzip --best" -cvf "${SCRIPTDIR}/../out/${krew_tar_archive}" ./*
```

Comment:
Definitely opt in for longer flag alias when you can, for readability.

Reply:
sure, good idea.